CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 **Internal**:
 
 - Emit a `service.back_pressure` metric that measures internal back pressure by service. ([#1583](https://github.com/getsentry/relay/pull/1583))
+- Scrub PII in the span description. ([#1590](https://github.com/getsentry/relay/pull/1590))
```

Comment:
```suggestion
- Always scrub PII in the span description. So far, the user had to configure this by hand. ([#1590](https://github.com/getsentry/relay/pull/1590))
```

Reply:
I'm not very consistent with this myself, but let's also add this to `py/CHANGELOG.md`. See https://github.com/getsentry/relay#instructions-for-changelogs.

Reply:
Will do